### DISCUSSION TITLE
Cross-reference pdf.js versions to Firefox versions

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [mozilla/pdf.js#17341] [Q&A]
Is there a table that cross-references pdf.js versions to Firefox versions?

**Comment by calixteman** [comment]
It's the best I can propose:
https://bugzilla.mozilla.org/buglist.cgi?query_format=advanced&short_desc_type=allwordssubstr&list_id=16805552&short_desc=update&resolution=FIXED&component=PDF%20Viewer&classification=Client%20Software&classification=Developer%20Infrastructure&classification=Components&classification=Server%20Software&classification=Other
For example, if you open:
https://bugzilla.mozilla.org/show_bug.cgi?id=1856144#c5
you'll see a field `target milestone` set to `120 branch`.

**Reply by Gitoffthelawn** [reply]
Thanks.  I appreciate it.  If an intern wants to make a table, it would be useful, but your link makes the information available with a little more effort.

BTW, does Bugzilla have a URL param to sort its output?  Once Bugzilla's data is loaded, one can click a header to sort, but pre-sorting with the newest items first would be a nice touch.
